src/compiler/crystal/command/docs.cr

```diff
@@ -4,7 +4,10 @@
 # is in `crystal/tools/doc/`
 
 class Crystal::Command
+  private VALID_OUTPUT_FORMATS = ["html", "json"]
```

Comment:
Pedantic: `%w(html json)`

Reply:
It's not really an improvement, just a different way to express this.